workorders.lic

```diff
@@ -598,6 +598,11 @@ class WorkOrders
     res.scan(/\d+/).first.to_i
   end
 
+  def deed_ingot_volume
```

Comment:
Was this change (and the next few) intentional? It doesn't seem related to the bankbot settings...

Reply:
I updated my copy of workorders to the current so no merge errors would appear.

Reply:
I updated my copy of workorders to the current so no merge errors would appear.

Reply:
I think you may have updated your copy directly instead of using git merge. It looks like git got a bit confused by this and didn't see a real merge. I tried to fix it, but there's still one line in `workorders` that looks off (I added a comment to it).